benchmarks_test.go

```diff
@@ -11,6 +12,8 @@ import (
 	"code.cloudfoundry.org/go-diodes"
 )
 
+var randData func(int) *[]byte = randDataGen()
```

Comment:
They type is actually inferred in init and can simply be `var randData = randDataGen()`

Reply:
thats fair, I'll fix it